data/json/monsters.json

```diff
@@ -6768,6 +6768,66 @@
     "fear_triggers":["FIRE"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_razorclaw",
+    "name": "razorclaw",
+    "species":"MUTANT",
+    "default_faction":"razorclaw",
+    "symbol":"Y",
+    "color":"magenta",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":20,
+    "aggression": 4,
+    "morale":100,
+    "speed": 85,
+    "melee_skill":6,
+    "melee_dice":4,
+    "melee_dice_sides":6,
+    "melee_cut":7,
+    "dodge":2,
+    "armor_bash": 10,
+    "armor_cut": 18,
+    "hp":80,
+    "vision_day": 30,
+    "vision_night":15,
+    "death_function":"NORMAL",
+    "special_attacks":[["SHRIEK_ALERT", 6], ["SHRIEK_STUN", 1]],
+    "description":"A man-sized crustacean clad in an iron-like chitin, constantly emiting the most horrible of shrieks.  Often spotted near shipwrecks or other dark damp places, which it uses as nesting grounds.",
```

Comment:
Constantly emiting shrieks? It only shrieks when its angry.

Also, "emitting", with two 't's